### ISSUE TITLE
Fix NoneType Error when calling def _transform_settings

### ISSUE THREAD

**Issue by grkamarko430** [dawillcox/pyBreezeChMS#4]
**I keep getting the following error when calling .list_funds():**

> line 125, in _transform_settings
>     return {k: _transform_setting(k, v) for k, v in args.items() if v}
>                                                     ^^^^^^^^^^
> AttributeError: 'NoneType' object has no attribute 'items'

**The following code fixed this error:**
```

def _transform_settings(args: Mapping) -> dict:
    """
    Given a mapping of parameter:value, return a dict with all of the parameters
    that actually had a value, but with values converted to API-compatible strings
    using _transform_setting().
    :param args: Mapping of parameter:value
    :return: New dict with values suitably transformed
    """
    if args is None:
        return {}
    return {k: _transform_setting(k, v) for k, v in args.items() if v}

```

**Comment by dawillcox** [comment]
Ewww.  You're right. A "trivial" case that the  unit test didn't catch. I'll work on that. It won't be a difficult fix, but my plate is a bit full right now.

As a workaround, if you call with include_totals=True, it will work. A bit of extra overhead on the server side, I'm sure, but at least it will work until this is fixed.

**Comment by dawillcox** [comment]
Fixed in release 1.4.2.
